data/data/agent/files/usr/local/bin/get-container-images.sh.template

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+. /usr/local/bin/release-image.sh
+
+ImagesEnvFile=${1}/agent-images.env
+AssistedEnvFile=/usr/local/share/assisted-service/assisted-service.env
```

Comment:
I'd prefer to create a new file rather than modify the one created by ignition. Can always pass more env files to podman.

Reply:
Sounds good, updated